src/pieces/commands.py

```diff
@@ -348,6 +349,40 @@ def check():
 ###############################################################################
 ############################## HELPER FUNCTIONS ###############################
 ###############################################################################
+def levenshtein_distance(s1, s2):
```

Comment:
probably the helper function should be in a separate file

Reply:
Sounds good. But this is a great implementation!

Reply:
Ty